examples/inverse/plot_rap_music.py

```diff
@@ -0,0 +1,54 @@
+"""
+=========================================
+Compute Rap-Music on evoked data
+=========================================
+
+Compute a Recursively Applied and Projected MUltiple Signal Classification
+(RAP-MUSIC) on evoked dataset.
+
+The reference for Rap-Music is:
+J.C. Mosher and R.M. Leahy. 1999. Source localization using recursively
+applied and projected (RAP) MUSIC. Trans. Sig. Proc. 47, 2
+(February 1999), 332-340.
+DOI=10.1109/78.740118 http://dx.doi.org/10.1109/78.740118
+"""
+
+# Author: Yousra Bekhti <yousra.bekhti@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+
+from mne.datasets import sample
+from mne.beamformer import rap_music
+from mne.io.pick import pick_types_evoked
```

Comment:
don't you have access to it in mne namespace? if so just use 

mne.pick_types_evoked